README.md

```diff
@@ -892,6 +892,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Samuel Z. Bouye](https://github.com/zamblebi)
   - [Samuele Simeone](https://github.com/S4muele)
   - [Samyak Jain](https://github.com/samyakjain10)
+  - [Samwel Chege](https://github.com/samwel-chege)
```

Comment:
Please add your name in line 894, after Samuele Simeone.
Let us know if you need any help with that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
Okay thanks let me do that now.